composer.json

```diff
@@ -13,7 +13,7 @@
         "php": "^5.6 || ^7.0",
         "ext-hash": "*",
         "ext-json": "*",
-        "ext-mongodb": "^1.6"
+        "ext-mongodb": "^1.7"
```

Comment:
Just confirming, but this works on the `master` branch of the driver because `phongo_version.h` already reports itself as `1.7.0-dev`, correct?